src/js/common/schemaform/helpers.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import { Route, IndexRedirect } from 'react-router';
+import FormPage from './FormPage';
+
+function createFormPageList(formConfig) {
+  return Object.keys(formConfig.chapters)
+    .reduce((acc, chapter) => {
```

Comment:
Accumulator. It's a sort of common generic name for that parameter in a reduce function, but I should find a more descriptive name.